lib/index.js

```diff
@@ -89,8 +98,16 @@
             query[key] = this._options.query[key];
         }
 
-        if (this._cookies.S) {
-            query.session_token = this._cookies.S
+        var sessionToken;
+
+        if ('getSessionToken' in this._http) {
```

Comment:
Nah, maybe I'm just being too cautious. A simple "does it exist" check is probably fine.